src/server/styles.test.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import styled, { ServerStyleSheet } from 'styled-components';
+import { renderToString } from 'react-dom/server';
+import { getStyleTag } from './styles'; // eslint-disable-line no-unused-vars
+
+// mock up and render a simple styled application
+const sheet = new ServerStyleSheet();
+const StyledHeading = styled.h1`
+  color: teal;
+`;
+const styleWithAttr = inlineAttribute => `<style ${inlineAttribute} data-reactroot="">
+/* sc-component-id: StyledHeading-av5ml9-0 */
+.StyledHeading-av5ml9-0 {} .gDvQnu{color:teal;}</style>`;
```

Comment:
Since you've used the StyledComponents outputted styles here, whenever `styled-components` updates their algorithm for generating classes, these unit tests will fail.

Reply:
Yes, they hash the CSS for the class names. 
I've seen changes in the order styles are ouptutted, in a minor update to `styled-components`. 
I'd be okay with keeping this as-is and then review as we pull in changes to styled-components.